spec/spec_helper.rb

```diff
@@ -4,10 +4,10 @@
 include Mercury::TestUtils
 
 # the block must return a Cps
-def test_with_mercury_cps(sources, queues, parallelism: 1, &block)
+def test_with_mercury_cps(sources, queues, **kws, &block)
```

Comment:
This change means that the default value for `parallelism` is no longer being set and `sync_spec` is not passing it in. If that default value should be set, perhaps pass `**({parallelism: 1}.merge(kws))` into the `.open` call.